### PR TITLE
Scraper

### PR DESCRIPTION
**Pull request by PRElias**
Hi,

I made some improvements and I would like to ask you if you agree with my vision. Included a script for easy scraper and a reg hack for executing script as admin by context menu.

Made tests on choco auto-install and in my opinion, the worst cenario will be the user having to re-execute.

### CODE REVIEW

**Comment by Francommit** [comment]
Hey @PRElias  been a bit busy this week with work. 
From your earlier work I suspect this should be great!

Is it ok if I postpone this to the weekend? Bit tied up at the moment!

Cheers.

File: prepare.ps1 [hunk 1]
```diff
@@ -750,16 +750,24 @@ Screensaver = 0
...
 Write-Output $dolphinConfigFileContent  > $dolphinConfigFile
 
+# 
+# 16. Fixing epsxe bug setting the registry
+# 
+$path = 'HKCU:\SOFTWARE\epsxe\config'
+
+New-Item -Path $path -Force | Out-Null
+
+Set-ItemProperty -Path $path -Name 'CPUOverclocking' -Value '10'
```
Comment (Francommit) [inline prepare.ps1:788]: Do you mind adding a URL comment here to where this is from? Will be good to refer back to later in time in-case it's fixed at some point.

File: prepare.ps1 [hunk 2]
```diff
@@ -565,14 +565,201 @@ New-Item -ItemType Directory -Force -Path $requiredTmpFolder
...
+[USBPassthrough]
+Devices = 
+[Sysconf]
+SensorBarPosition = 1
+SensorBarSensitivity = 50331648
+SpeakerVolume = 88
+WiimoteMotor = True
+WiiLanguage = 1
+AspectRatio = 1
+Screensaver = 0
```
Comment (Francommit) [inline prepare.ps1:773]: This is great. It's exactly the type of stuff I wanted to be in here 👍

**Comment by Francommit** [comment]
Ok I had a quick glance over and it looks good! I'm going to run it on my machine over the weekend. Cheers!

File: prepare.ps1 (hunk 1 above)
Comment (PRElias) [reply prepare.ps1:788]: Ok. Included as comment

**Comment by PRElias** [comment]
No hurry, take your time. It's been a pleasure working on this project. Thank you!

**Review by Francommit — APPROVED** [review]

**Comment by Francommit** [comment]
Thanks @PRElias  for the great additions!

**Pull request merged**